scripts/extensions/sams/src/store/assets/actions.ts

```diff
@@ -160,6 +163,37 @@ export function editAsset(assetId?: string): IAssetActionTypes {
     };
 }
 
+export function unlockAsset(asset: IAssetItem): IThunkAction<void> {
+    return (dispatch) => {
+        return samsApi.assets.unlockAsset(asset, {})
+            .then(() => {
+                return dispatch(queryAssetsFromCurrentSearch(LIST_ACTION.REPLACE));
```

Comment:
Reloading the lists would be overkill here. It would be more efficient to have a separate action/reducer to update a single Asset in the redux store (using the updated attributes from the response to the `lock` or `unlock` api call)